Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -97,6 +97,7 @@
   requiredTechnologies:
   - BasicResearch
   unlockedRecipes:
+  - Vape
```

Comment:
kinda a nitpick but at least put it in biological tech or somewhere there. it isn't really chemistry equipment.